hrm/department_app/tests/test_serializers.py

```diff
@@ -0,0 +1,77 @@
+from rest_framework.test import APITestCase
```

Comment:
The tests below do NOT make sense because you are testing Django logic. We only need to test our custom logic. We have the credit of trusting third-party libraries, so we shouldn't test them. You can just remove them.

Reply:
I already fixed it.